distributions/aws/examples/rds-s3/README.md

```diff
@@ -0,0 +1,136 @@
+
+```
+export S3_BUCKET=<YOUR_S3_BUCKET_NAME>
+export CLUSTER_REGION=<YOUR_CLUSTER_REGION>
+aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
+```
+
+5. Create RDS Instance
+
+Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. 
+
+
+### 2. Configure Kubeflow Pipelines
+
+1. Configure the following files in `distributions/aws/apps/pipelines` directory:
+
+    1. Configure `distributions/aws/apps/pipelines/params.env` file with the RDS endpoint URL, S3 bucket name, and S3 bucket region that were configured when following the steps in Create RDS Instance and Create S3 Bucket steps in prerequisites(#1-prerequisites). 
+        - For example, if your RDS endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, S3 bucket name is `kf-aws-demo-bucket`, and s3 bucket region is `us-west-2` your `params.env` file should look like:
+        - ```
+          dbHost=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
+
+          bucketName=kf-aws-demo-bucket
+          minioServiceHost=s3.amazonaws.com
+          minioServiceRegion=us-west-2
+          ```
+    1. Configure `distributions/aws/apps/pipelines/secret.env` file with your RDS database username and password that were configured when following the steps in Create RDS Instance.
+        - For example, if your username is `admin` and your password is `Kubefl0w` then your `secret.env` file should look like:
+        - ```
+          username=admin
+          password=Kubefl0w
+          ```
+    1. Configure `distributions/aws/apps/pipelines/minio-artifact-secret-patch.env` file with your AWS credentials. These need to be long term credentials from an IAM user and not temporary. 
+        - Find more details about configuring/getting your AWS credentials here:
+        https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html
+        - ```
+          accesskey=AXXXXXXXXXXXXXXXXXX6
+          secretkey=eXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXq
+          ```
+
+### 3. Configure Katib
+
+
+1. Configure the `distributions/aws/apps/katib-external-db-with-kubeflow/secrets.env` file with the RDS DB name, RDS endpoint URL, RDS DB port, and RDS DB credentials that were configured when following the steps in Create RDS Instance.
+    - For example, if your database name is `KubeflowRDS`, your endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, your DB port is `3306`, your DB username is `admin`, and your DB password is `Kubefl0w` your `secrets.env` file should look like:
+    - ```
+      KATIB_MYSQL_DB_DATABASE=kubeflow
+      KATIB_MYSQL_DB_HOST=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
+      KATIB_MYSQL_DB_PORT=3306
+      DB_USER=admin
+      DB_PASSWORD=Kubefl0w
+      ```
+
+
+### 4. Install using the following command:
```

Comment:
Can we say explicitly what are we installing here? 
Is it complete Kubeflow deployment or just RDS + S3 part?